scripts/.tests-compose.yaml

```diff
@@ -4,7 +4,8 @@ services:
     image: broadinstitute/clojure-node
     volumes:
       - work:/w
-      - node-modules:/w/node_modules
+######## Not needed for unit tests
+#      - node-modules:/w/node_modules
```

Comment:
Let's take these out rather than comment. The passing tests prove that they are not needed.